Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -1378,7 +1379,9 @@ bool WebFrame::handleContextMenuEvent(const PlatformMouseEvent& platformMouseEve
     bool handled = frame->eventHandler().sendContextMenuEvent(platformMouseEvent);
 #if ENABLE(CONTEXT_MENUS)
     if (handled)
-        protectedPage()->protectedContextMenu()->show();
+        if (protectedPage()->protectedContextMenu()->show())
```

Comment:
You can combine the two if statements.
```suggestion
        if (handled && protectedPage()->protectedContextMenu()->show())
```